README.md

```diff
@@ -86,6 +81,68 @@ Additional annotation information and external identifiers for Human-GEM reactio
 * Save using the `exportModel.m` function (from [RAVEN Toolbox](https://github.com/SysBioChalmers/RAVEN))
 
 
+## Reaction, Metabolite, and Gene Annotations
+
+Additional annotation information and external identifiers for Human-GEM reactions, metabolites, and genes are provided as `tsv` files in the `model/` directory. The structure of the `tsv` files are tabulated below.
+
+* `reactions.tsv` content:
+
+filedname      |  annotation             |
+---------------|------------------------ |
+rxns           |identical to `model.rxns`|
+rxnKEGGID      |KEGG reaction ID        |
+rxnBiGGID      |BiGG reaction ID        |
+rxnEHMNID      |EHMN reaction ID        |
+rxnHepatoNET1ID|HepatoNET1 reaction ID  |
+rxnREACTOMEID  |REACTOME ID             |
+rxnRecon3DID   |Recon3D reaction ID     |
+rxnMetaNetXID  |MetaNetX reaction ID    |
+rxnHMR2ID      |HMR2 reaction ID        |
+rxnRatconID    |Ratcon reaction ID      |
+rxnTCDBID      |TCDB ID                 |
+spontaneous    |Spontaneous status      | 
+rxnMAID        |MA reaction ID          |
+rxnRheaID      |Rhea ID                 |
+rxnRheaMasterID|Master Rhea ID          |
+
+
+* `metes.tsv` content:
+
+filedname      |  annotation             |
```

Comment:
is it supposed to be filedname, and not filename? Also in the table further down

Reply:
it should be `fieldname `